ragnarson-stylecheck.gemspec

```diff
@@ -1,15 +1,15 @@
 Gem::Specification.new do |s|
   s.name          = "ragnarson-stylecheck"
-  s.version       = "0.8.2"
+  s.version       = "1.0.0"
   s.summary       = "Automatic style check for ragnarson projects"
   s.description   = "Wraps rubocop for simple and consisten experience"
-  s.authors       = ["Grzesiek Kołodziejczyk", "Maciej Małecki", "Oskar Szrajer", "Piotr Marciniak"]
+  s.authors       = ["Grzesiek Kołodziejczyk", "Maciej Małecki", "Oskar Szrajer", "Piotr Marciniak", "Michał Gołdys"]
   s.email         = "hello@ragnarson.com"
   s.files         = Dir["{config,lib}/**/*", "MIT-LICENSE", "README.md", "./"]
   s.homepage      = "https://github.com/ragnarson/ragnarson-stylecheck"
   s.license       = "MIT"
 
   s.add_development_dependency "bundler", "~> 1.10"
   s.add_dependency "rake", ">= 10.0"
-  s.add_dependency "rubocop", "~> 0.64"
+  s.add_dependency "rubocop", "~> 0.93.1"
```

Comment:
Why not the latest stable release? 1.18.3? Are there too many breaking changes?

Reply:
I have bumped rubocop version to meet requirements of Cookstyle gem that we will be migrating to due to deprecation of Foodcritic. 
But you're right, I've already bumped rubocop version to latest one, and pushed changes.
We could use higher version of Cookstyle.

Reply:
One of The Good Scout rules says that if you're about to bump any version of any dependency (chef, gem, or whatever) you should always aim for the latest stable version available. Unless you see some potential problems (e.g. with API incompatibility, refactor) that you don't have time to deal with. 

If you can't do it. A good next action would be to leave a comment about those potential issues in the code. Ideally, if you can't upgrade to the latest stable release, you should create a task ticket for doing it - and describe all the findings you already gather.